CI/setup-windows-sdk.sh

```diff
@@ -284,4 +284,12 @@ cd ~ || exit 1
 echo "  ... setup-windows-sdk.sh shell script finished."
 echo ""
 
+echo "Copy the following lines into the build environment for a project in Qt Creator:"
+echo "See https://doc.qt.io/qtcreator/creator-how-set-project-environment.html#change-the-environment-for-a-project"
+echo ""
+MSYS_ROOT=$(cygpath -aw /)
+echo "MINGW_BASE_DIR=${MSYS_ROOT}$(echo ${MSYSTEM_PREFIX} | sed 's/\//\\/g')"
```

Comment:
```suggestion
echo "MINGW_BASE_DIR=$(cygpath -am "${MSYSTEM_PREFIX}")"
```

Reply:
Have you verified that this suggestion works? I tried cygpath earlier, and its output was not working. Only this way worked.